src/background/messaging.ts

```diff
@@ -125,3 +128,15 @@ export const registerHandlerListener = async (message: any) =>
     new Promise<unknown>((resolve) => {
         responseHandlers[message.messageId] = resolve;
     });
+export const setAuthTokenHandler = async (message: any) => {
+    await browser.storage.local.set({ point_token: message.token });
+    return { ok: true };
+};
+
+export const getAuthToken = async () => {
+    const { point_token } = await browser.storage.local.get("point_token");
```

Comment:
Browser local storage is available for any website. Does this means that anyone has access to the `point_token`? If so, are you sure it is safe to let them know it?

Reply:
This is local storage for extensions. It isn't accessible to web pages. Relevant sources:
* https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/local
* https://stackoverflow.com/questions/53245061/are-there-any-restrictions-on-access-to-browser-extensions-storage